.travis.yml

```diff
@@ -20,9 +20,9 @@ env:
     - DB=mysql
     - DB=pgsql
     - DB=sqlite
-    - DB=mysql CORE_BRANCH=stable19
-    - DB=pgsql CORE_BRANCH=stable19
-    - DB=sqlite CORE_BRANCH=stable19
```

Comment:
Don't delete NC19.. Just add NC20.

Currently the stable versions are NC18 to NC20, so we must test all 3. :wink: